Cargo.toml

```diff
@@ -29,9 +32,20 @@ path = "crossbeam-ebr/crossbeam-epoch"
 package = "crossbeam-pebr-epoch"
 path = "crossbeam-pebr/crossbeam-epoch"
 
-# TODO: remove this
 [profile.release]
+lto = true
+codegen-units = 1
+panic = "abort"
```

Comment:
why?

Reply:
* LTO & codegen-units: to maximize the performance. The LTO was significant for the CIRC experiments.
* abort on panic: I used it to prevent *panic while panic* problem while developing, but it seems it is not necessary for now. I have removed it.